source/docs/02_api_libraries/02_initialization/01_initialization.py.md

```diff
@@ -4,6 +4,7 @@ After [installing the GoCardless Python library](#installation), you will need t
 
 ```python
 import gocardless
+gocardless.environment = 'sandbox'
 gocardless.set_details(app_id="DUMMY_APP",
         app_secret="INSERT_APP_SECRET_HERE",
         access_token="INSERT_MERCHANT_ACCESS_TOKEN",
```

Comment:
Hey thanks for the PR. Could maybe add a note about the environment below this block instead:

`By default the library will attempt to use the GoCardless production environment, for testing purposes this is not what you want and you should set the gocardless.environment to "sandbox":`

```
gocardless.environment = "sandbox"
```